pkg/kubectl/cmd/get/get.go

```diff
@@ -160,18 +165,34 @@ func NewCmdGet(parent string, f cmdutil.Factory, streams genericclioptions.IOStr
 		Long:                  getLong + "\n\n" + cmdutil.SuggestAPIResources(parent),
 		Example:               getExample,
 		Run: func(cmd *cobra.Command, args []string) {
-			cmdutil.CheckErr(o.Complete(f, cmd, args))
-			cmdutil.CheckErr(o.Validate(cmd))
-			cmdutil.CheckErr(o.Run(f, cmd, args))
+			cmdutil.CheckErr(cmd.RunE(cmd, args))
 		},
-		SuggestFor: []string{"list", "ps"},
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
We should factor out parts of `Run` you need to unit test and test that. I'd rather leave the main parts as they were.

Reply:
It is even testing setting the flags and all the other unit tests test the whole method so this would essentially mean creating and identical function and make the real function a dummy wrapper.

Shouldn't the whole kubectl move to `RunE` eventually? This seems like a first step in backwards compatible way.